### PR TITLE
FAQ Wip

### PR DESCRIPTION
**Pull request by RachelAnnB**
-added content, styles, and media queries to the faq section
-fixed some spacing issues on a few other files

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -28,8 +28,34 @@ $('.responsive').slick({
...
\ No newline at end of file
+		// You can unslick at a given breakpoint now by adding:
+		// settings: "unslick"
+		// instead of a settings object
+		]
+});
+
+//---------------Show/Hide FAQs---------------//
+function showonlyone(thechosenone) {
+	var newboxes = document.getElementsByTagName("div");
```
Comment (bencallahan) [inline src/js/index.js]: Seems aggressive to get all the `div`s in the entire document. Could you scope this back so you are only needing to loop through the proper elements?

File: src/partials/_faq.hbs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <h2 class="faq__heading">FAQ</h2>
+
+  <div class="faq__list__links">
+    <a class="faq__link" href="javascript:showonlyone('newboxes1');">Retailers</a>
+    <a class="faq__link" href="javascript:showonlyone('newboxes2');">Parents</a>
+  </div>
+
+  <div class="newboxes" id="newboxes1" style="display: block;padding: 5px;">
+    <div class="list1">
+      <section class="faq__box">
```
Comment (bencallahan) [inline src/partials/_faq.hbs]: Regarding your markup choices here, another option would be to use a `<dl>` with `<dt>` as the question and `<dd>` as the answer. Nothing necessarily wrong with using `<div>`s and `<sections>`s. Just something to explore!

File: src/scss/_carousel.scss [hunk 3]
```diff
@@ -1,97 +1,97 @@
...
 
 .next {
-	color: #999;
-	position: absolute;
-	top: 38%;
-	right: -2em;
-	font-size: 1.5em;
-	:hover {
-		cursor: pointer;
-		color: black;
```
Comment (bencallahan) [inline src/scss/_carousel.scss:89]: Awesome to see you taking the time to clean up the spacing! Well done.

File: src/scss/_faq.scss [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  color: $color-tan;
+  text-decoration: none;
+  padding: 0.5em; //8px
+  border: 2px solid $color-brown;
+
+  &:hover {
+    background-color: $color-brown;
+  }
+  
+  &:active {
```
Comment (bencallahan) [inline src/scss/_faq.scss]: Could you consolidate this with the selector above like so:

`&:hover, &:active {`

Might tighten up the code a bit.

File: src/js/index.js (hunk 1 above)
Comment (RachelAnnB) [inline src/js/index.js]: Good point, I'll try document.getElementsByClassName instead.

File: src/partials/_faq.hbs (hunk 2 above)
Comment (RachelAnnB) [inline src/partials/_faq.hbs]: I've never used those tags before, but I'll definitely look into it!

**Pull request merged**